README.rst

```diff
@@ -1,3 +1,11 @@
+.. image:: https://img.shields.io/pypi/v/pytest-localserver.svg?style=flat
+    :alt: PyPI Version
+    :target: https://pypi.python.org/pypi/pytest-django
+
+.. image:: https://img.shields.io/pypi/pyversions/pytest-localserver.svg
+    :alt: Supported Python versions
+    :target: https://pypi.python.org/pypi/pytest-django
```

Comment:
Django? :wink: I'm guessing these URLs are meant to point at pytest-localserver instead.

Reply:
Ah, you are right. It was a copy from `pytest-django` :smile: 
I've fixed it!